build/images/Dockerfile.simulator.build.ubuntu

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.15 as antrea-build
```

Comment:
Since we can cross-compile on the host machine directly no matter the OS it is on, I actually wonder if we still need these builder images. @tnqn, @antoninbas do you have any thought about this?

Reply:
It's important to have a docker build which does not require any dependency besides docker IMO (e.g. the correct version of the go compiler). As far as I'm concerned, Dockerfile.build.ubuntu is our "main" Dockerfile. It is the one we use in CI and I personally never use the other one (Dockerfile.ubuntu), which assumes that the binaries have already been built locally. However, I do understand that Dockerfile.ubuntu is also important for dev workflow, to build new images faster.

That being said, it's likely that having both for the simulator is an overkill. Maybe we should agree on a single build workflow for the simulator.